app.rb

```diff
@@ -1,5 +1,96 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/game'
+require './lib/player'
+
 class RockPaperScissors < Sinatra::Base
+
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
+    erb(:index)
+  end
+
+  post '/players' do
+    p params
+    @player_one = Player.new(params[:player_one_name], params[:player_one_CPU])
+    @player_two = Player.new(params[:player_two_name], params[:player_two_CPU])
+    @spock_game = params[:spock_game]
+    $game = Game.new(@player_one, @player_two, @spock_game)
+    if $game.spock_game?
```

Comment:
Great that you did the spock-lizard extension. The if-else could be broken down to?

redirect '/spock-moves' if $game.spock_game?
redirect '/moves'